app/event/send-manual-ledger-review-event.js

```diff
@@ -13,4 +28,19 @@ const sendManualLedgerReviewEvent = async (paymentRequestId, user, status) => {
   await raiseEvent(event)
 }
 
+const sendV2ManualLedgerReviewEvent = async (paymentRequestId, user, status) => {
+  const paymentRequest = await getPaymentRequestByRequestId(paymentRequestId)
+  const event = {
+    source: SOURCE,
+    type: `${LEDGER_ASSIGNMENT_QUALITY_CHECK}${status.toLowerCase()}`,
```

Comment:
@johnwatson484 is the `.` not required because of the formatting of this `type`?

Reply:
Yes, you are correct.  Apologies.